### PR TITLE
Dev v1.1.1

### PR DESCRIPTION
**Pull request by danzelenak-usgs**
Would like to merge this into master and tag the version number.

### CODE REVIEW

File: scripts/espa_reprojection.py [hunk 1]
```diff
@@ -1253,7 +1253,32 @@ def convert_imageXY_to_mapXY(image_x, image_y, transform):
...
+                        'st_thermal_radiance_band11'],
+    'st': ['st'],
+    'Rrs': ['rrs_band1',
+            'rrs_band2',
+            'rrs_band3',
+            'rrs_band4',
+            'rrs_band5',
+            'rrs_band6',
+            'rrs_band6',
+            'chor_a']
```
Comment (gschmidt-usgs) [inline scripts/espa_reprojection.py]: This 'chor_a' should be 'chlor_a'.

Comment (danzelenak-usgs) [reply scripts/espa_reprojection.py]: fixed

File: scripts/espa_reprojection.py [hunk 2]
```diff
@@ -1660,10 +1770,63 @@ def main():
...
+            # Extents will be in the same projection as the output projection.
+            base_warp_command.extend(['-te'])
+            base_warp_command.extend(extents)
+            logger.debug('Updated CMDLINE: {}'.format(base_warp_command))
+
     # Update the XML to reflect the new warped output
     update_espa_xml(args, espa_metadata)
 
     # Validate the XML
+    print(espa_metadata)
```
Comment (gschmidt-usgs) [inline scripts/espa_reprojection.py]: This was just a debugging line.  It should probably be removed to keep the output cleaner.

Comment (danzelenak-usgs) [reply scripts/espa_reprojection.py]: fixed

**Pull request merged**